src/main/java/org/mockito/internal/debugging/Java9PlusLocationImpl.java

```diff
@@ -1,304 +0,0 @@
-/*
- * Copyright (c) 2007 Mockito contributors
- * This program is made available under the terms of the MIT License.
- */
-package org.mockito.internal.debugging;
-
-import org.mockito.exceptions.base.MockitoException;
-import org.mockito.exceptions.stacktrace.StackTraceCleaner;
-import org.mockito.exceptions.stacktrace.StackTraceCleaner.StackFrameMetadata;
-import org.mockito.internal.configuration.plugins.Plugins;
-import org.mockito.internal.exceptions.stacktrace.DefaultStackTraceCleaner;
-import org.mockito.invocation.Location;
-
-import java.io.Serializable;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
-import java.util.function.Function;
-import java.util.function.Predicate;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-class Java9PlusLocationImpl implements Location, Serializable {
```

Comment:
The implementation of this class is different than what we still have in `LocationImpl`. Shouldn't we update `LocationImpl` to use this implementation? E.g. delete `LocationImpl` and rename this file to it.

Reply:
@TimvdLippe this is correct, the implementation is indeed different: `Java9PlusLocationImpl ` uses refection for referencing `StackWalker` APIs, this is not needed anymore because `LocationImpl` uses regular imports instead.

Reply:
Ah of course. Thanks for the clarification!